taxi/tests/test_models.py

```diff
@@ -0,0 +1,41 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.models import Manufacturer, Driver, Car
+
+
+class TestsModels(TestCase):
+    def test_manufacturer_str(self):
+        manufacturer = Manufacturer.objects.create(name="test", country="Test")
+        self.assertEqual(str(manufacturer), f"{manufacturer.name} {manufacturer.country}")
+
+    def test_driver_str(self):
+        driver = Driver.objects.create(
+            id="1",
+            username="testname",
+            first_name="first_test",
+            last_name="last_test",
+            license_number="HFJ12341"
+        )
+        self.assertEqual(str(driver), f"{driver.username} ({driver.first_name} {driver.last_name})")
+
+    def test_car_str(self):
+        manufacturer = Manufacturer.objects.create(name="Test Manufacturer")

```

Comment:
The `Manufacturer` model creation is missing the `country` field, which might be required based on your model definition. Ensure that all required fields are provided during object creation to avoid potential errors.